src/helpers/i18n.js

```diff
@@ -27,9 +27,11 @@ export function formatAmount(intl, amount) {
 }
 
 export function formatDateFromISO(mm, intl, date) {
-  let isSecondaryCalendar = JSON.parse(localStorage.getItem('isSecondaryCalendarEnabled'));
+  const isSecondaryCalendar = JSON.parse(localStorage.getItem("isSecondaryCalendarEnabled"));
```

Comment:
"isSecondaryCalendarEnabled" could be moved to consts:
`
const STORAGE_KEY_SECONDARY_CALENDAR = "isSecondaryCalendarEnabled";
`